languages/javascript/reference/info/scope.md

```diff
@@ -0,0 +1,7 @@
+## Scope / Visibility
+
+Most languages have some concept of encapsulation through scope and visibility.
+Tracks should explain the language-concepts and keywords that their track uses (e.g. `public`, `private`, `export`) and what the normal way of using those scopes are in their language. An exercise may be appropriate, but providing examples in stubs early on may avoid overwhelming the student.
+
+[Global Scope](#global-scope)
```

Comment:
I would also make this a sub headline (it will get this hash-automagically on GitHub).

Reply:
What do you mean?

Reply:
Just write

```md
## Global Scope
Something is in the global scope in JavaScript when it is not in any local scope and therefore accessible from everywhere. 
```

(Or something better than I just did.)

Then you should be able to link to it via `#global-scope`.

Reply:
Oh cool didn't know that, thanks.